sky-api-tutorial-auth-code-c-sharp.csproj

```diff
@@ -21,9 +20,10 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="1.0.4" />
```

Comment:
Should these all be updated? This package has version 2.2.0 out now.

Reply:
I was wondering about that too. Let me upgrade them as well.

Reply:
I would just go ahead and update everything to the latest that is supported by .NET 3.1. You can check on nuget.org if you expand dependencies on the latest version. Only thing to watch for is that you don't want a version that only supports 5.0, so you'd have to fall back to the previous major version of any library like that.

Reply:
Updated the packages